src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/TemplateCollectionFunctionalTests.cs

```diff
@@ -37,6 +37,7 @@ public class TemplateCollectionFunctionalTests : IAsyncLifetime
         private readonly string _defaultHl7v2TemplateImageReference = "microsofthealth/hl7v2templates:default";
         private readonly string _defaultCcdaTemplateImageReference = "microsofthealth/ccdatemplates:default";
         private readonly string _defaultJsonTemplateImageReference = "microsofthealth/jsontemplates:default";
+        private readonly string _defaultStu3ToR4TemplateImageReference = "microsofthealth/stu3tor4templates:default";
```

Comment:
Just curious have we pushed this image already?

Reply:
This reference represents the build-in template zip package.

Reply:
Yes, just want to make sure that we publish this default package when we release stu3 to r4 conversion.